dask/array/core.py

```diff
@@ -4122,7 +4122,7 @@ def asarray(a, **kwargs):
     elif type(a).__module__.split(".")[0] == "xarray" and hasattr(a, "data"):
         return asarray(a.data)
     elif isinstance(a, (list, tuple)) and any(isinstance(i, Array) for i in a):
-        return stack(a)
+        return stack(a, allow_unknown_chunksizes=allow_unknown_chunksizes, **kwargs)
```

Comment:
I don't think we should be passing the `kwargs` into this stack call.

```suggestion
        return stack(a, allow_unknown_chunksizes=allow_unknown_chunksizes)
```

Reply:
I agree.  Thanks.